src/EntityRevisionTrait.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\EntityRevisionTrait.
+ */
+
+namespace Drupal\entity;
+
+/**
+ * Provides fixes for revisions
+ */
+trait EntityRevisionTrait {
```

Comment:
Well, the only reason why I provide them here is to be able to call them cleanly and the IDE knows about them.